thefuck/rules/remove_shell_prompt_literal.py

```diff
@@ -0,0 +1,25 @@
+"""Fixes error for commands containing the shell prompt symbol '$'.
+
+This usually happens when commands are copied from documentations
+including them in their code blocks.
+
+Example:
+> $ git clone https://github.com/nvbn/thefuck.git
+bash: $: command not found...
+"""
+
+import re
+
+
+def match(command):
+    return (
+        "$: command not found" in command.output
+        and re.search(r"^[\s]*\$ [\S]+", command.script) is not None
+    )
+
+
+def get_new_command(command):
+    return command.script.replace("$", "", 1).strip()
+
+
+requires_output = True
```

Comment:
This can be omitted as [`requires_output`'s default value is `True`](https://github.com/nvbn/thefuck/blob/70b414aca2acbbac2cd3c75bc4c3e2dd489b3144/thefuck/types.py#L147).